git-importer.py

```diff
@@ -374,6 +368,21 @@ def merge(
     def merge_abort(self):
         self.repo.state_cleanup()
 
+    def last_commit(self):
+        try:
+            return self.repo.head.target
+        except:
+            return None
+
+    def clean(self):
+        for path, _ in self.repo.status().items():
+            logging.debug(f"Cleaning {path}")
+            try:
+                (self.path / path).unlink()
+                self.repo.index.remove(path)
+            except Exception:
```

Comment:
I take a PIP and throw it at you :)

Reply:
Yeah ... Added a logging.warning, and continue